public/images/list-green.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="39" height="39" viewBox="0 0 39 39" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M0 27.8571H39V22.2857H0V27.8571ZM0 39H39V33.4286H0V39ZM0 16.7143H39V11.1429H0V16.7143ZM0 0V5.57143H39V0H0Z" fill="#2B7A78"/>
```

Comment:
If the difference between icons is in color only then try to apply ‘currentColor’ and use only one file

Reply:
will try to implement it in separate pull request for grid/list feature. Thanks for advice!